apex/contrib/optimizers/distributed_fused_lamb.py

```diff
@@ -683,6 +687,7 @@ def _pipeline_block_reductions(self, block_id):
                     # Since the packed format is contiguous after reductions, only one norm is needed
                     l2_grad_norm_sq = torch.empty([1], device='cuda')
                     if 0:#self._full_ar:
+                        #TODO 
```

Comment:
Is this PR ready for merge?

Reply:
I knew Burc was working on this case (line 690), so the #1264 addresses the missing if branch